code/datalab-app/src/app/services/graphQLService.js

```diff
@@ -0,0 +1,32 @@
+import request from 'superagent-bluebird-promise';
```

Comment:
Not sure that we want to use superagent-bluebird-promise. Maybe fetch or axiom instead. Fine for this PR but lets discuss before adding more. We probably also want to isolate the HTTP library and wrap with our own service to give us flexibility to replace if we want.